app/views/sessions/new.html.erb

```diff
@@ -0,0 +1 @@
+<%= button_to "Log In With Spotify", spotify_oath_path, method: :get %>
```

Comment:
hmm Oath? or Oauth?

Reply:
> hmm Oath? or Oauth?

Thats a nice catch, maybe we should make a note on the project board and hotfix it with the next branch?